app/uk/gov/hmrc/eoricommoncomponent/frontend/views/match_nino_row_individual.scala.html

```diff
@@ -23,9 +23,12 @@
 @import uk.gov.hmrc.govukfrontend.views.Aliases.Button
 @import uk.gov.hmrc.govukfrontend.views.Aliases.Text
 
+
+
 @this(layout_di: layout, inputRadioGroup: helpers.inputRadioGroup, govukButton: GovukButton, errorSummary: helpers.errorSummary)
 
-@(matchNinoIdForm: Form[NinoMatchModel], service: Service)(implicit request: Request[_], messages: Messages)
+
+@(matchNinoIdForm: Form[NinoMatchModel]  ,service: Service)(implicit request: Request[_], messages: Messages)
```

Comment:
Very minor but may as well mention it since the other change is going to be made. Can the comma be moved back - just looks a bit odd